samples/highcharts/data-tools/googlesheets-html/demo.css

```diff
@@ -0,0 +1,41 @@
+:root {
+    --color-background: #fff;
+    --color-foreground: #000;
```

Comment:
As I see it is used by the `color` param which means text. I suggest to use a name like `--color-text` or something similar which is more clear.